bin/compile

```diff
@@ -362,6 +369,12 @@ trap clearGitCredHelper INT TERM EXIT
 
 determineTool
 
+ver_major=0
```

Comment:
Please use goVerMajor, goVerMinor, goVerPatch, goVerSpecial as it's more consistent with the rest of the variables in the buildpack.